scapy/layers/bluetooth.py

```diff
@@ -312,6 +312,109 @@ class SM_Master_Identification(Packet):
     fields_desc = [ XLEShortField("ediv", 0),
                     StrFixedLenField("rand", '\x00' * 8, 8), ]
 
+
+class EIR_Hdr(Packet):
+    name = "EIR Header"
+    fields_desc = [
+        FieldLenField("len", 0, fmt="B"),
```

Comment:
Hmm not sure... When building a packet, it should get set by the `EIR_<type_here>` packet after it. Using a `EIR_Hdr` by itself doesn't make a lot of sense.